services/forward-interop/test/service.test.js

```diff
@@ -22,6 +22,7 @@ let m2 = interop.InteropMessage.encode({
 test('telemetry data is forwarded to interop-proxy', async () => {
   let service = new Service({
     port: 4000,
+    uploadInterval: 300,
```

Comment:
I'd leave the existing tests the same, and then add a test which checks that the upload time can be decreased by half or something and that the test works as intended in the same way. (Decreasing is better than increasing since it makes the tests run faster.)